### ISSUE TITLE
CSAT Widget `aria-invalid` without form submission

### ISSUE THREAD

**Issue by danielclubb** [springernature/frontend-toolkits#865]
Hi team,

We think we have found an issue with the CSAT widget implementation.

If we provide an `error` property in the json config data required to use the widget, we get an accessibility error because the CSAT form has an `aria-invalid="true"` property by default, without the form having been submitted.

If we remove the `error` property from the json config data, then the accessibility error goes away, however, the error state is then broken as it does not have an error message to display.

We think this is happening because the template logic of the CSAT component looks for `error` in the context in order to add/remove the `aria-invalid="true"` attribute, and when this is provided in the json data, the attribute is added by default. 

If the property were to be renamed to `errorMessage` or something similar that isn't just `error` we think this would fix it.

Thanks!

CC: Andreas Koscinski <[andreas.koscinski@springernature.com](mailto:andreas.koscinski@springernature.com)>, Daniel Clubb <[daniel.clubb@springernature.com](mailto:daniel.clubb@springernature.com)>, Morgan Cugerone <[morgan.cugerone@springernature.com](mailto:morgan.cugerone@springernature.com)>
